backend/src/main/java/com/sscatalog/specialistsservicescatalog/services/ServicesService.java

```diff
@@ -0,0 +1,30 @@
+package com.sscatalog.specialistsservicescatalog.services;
+
+import com.sscatalog.specialistsservicescatalog.dtos.ServiceDto;
+import com.sscatalog.specialistsservicescatalog.entities.Service;
+import com.sscatalog.specialistsservicescatalog.repositories.ServiceRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@org.springframework.stereotype.Service
```

Comment:
Why do not you use just @Service annotation here ?

Reply:
Because my entity class for service has the same name